proto/commonfate/access/v1alpha1/access_request.proto

```diff
@@ -44,6 +45,11 @@ message QueryAccessRequestsRequest {
   string page_token = 1;
   bool archived = 2;
   optional commonfate.entity.v1alpha1.Order order = 3;
+  optional commonfate.entity.v1alpha1.EID requested_by = 4;
+  optional commonfate.entity.v1alpha1.EID closed_by = 5;
+  optional commonfate.entity.v1alpha1.EID approved_by = 6;
+  GrantStatus grant_status = 7;
```

Comment:
Could you please change this one to request status

The request status and grant status are different. A request may have grants with multiple statuses, at some point we may add something like granted_statuses to filter requests by the grants that they have.
for now though, I think the requests statuses active,pending, closed are what we need here
```suggestion
  GrantStatus grant_status = 7;
```